### ISSUE TITLE
export ScriptModule models to ONNX format failed

### ISSUE THREAD

**Issue by deepindeed** [pytorch/pytorch#35291]
## 🐛 Bug

Unable to export ScriptModule models using torch.onnx.export interface to onnx model

## To Reproduce

> Steps to reproduce the behavior:

Runing the following example script:
```
import torch
import torchvision
m = torchvision.models.resnet50()
x = torch.rand((1, 3, 224, 224))
traced_m = torch.jit.trace(m, x)
f = 'model.pt'
torch.jit.save(traced_m, f)
loaded_m = torch.jit.load(f)
#torch.onnx.export(loaded_m, x, 'model.onnx')
torch.onnx._export(loaded_m, x, 'model.onnx', example_outputs=torch.rand((1, 1000)))
```

then, the following error occurs:

```
dim  <class 'NoneType'> start_dim  <class 'int'> end_dim  <class 'int'>
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "/usr/local/lib/python3.6/dist-packages/torch/onnx/__init__.py", line 26, in _export
    result = utils._export(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/torch/onnx/utils.py", line 416, in _export
    fixed_batch_size=fixed_batch_size)
  File "/usr/local/lib/python3.6/dist-packages/torch/onnx/utils.py", line 296, in _model_to_graph
    fixed_batch_size=fixed_batch_size, params_dict=params_dict)
  File "/usr/local/lib/python3.6/dist-packages/torch/onnx/utils.py", line 135, in _optimize_graph
    graph = torch._C._jit_pass_onnx(graph, operator_export_type)
  File "/usr/local/lib/python3.6/dist-packages/torch/onnx/__init__.py", line 179, in _run_symbolic_function
    return utils._run_symbolic_function(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/torch/onnx/utils.py", line 657, in _run_symbolic_function
    return op_fn(g, *inputs, **attrs)
  File "/usr/local/lib/python3.6/dist-packages/torch/onnx/symbolic_helper.py", line 129, in wrapper
    return fn(g, *args)
  File "/usr/local/lib/python3.6/dist-packages/torch/onnx/symbolic_opset9.py", line 1712, in flatten
    end_dim = dim + end_dim
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int' (occurred when translating flatten)

```
## Expected behavior

export ScriptModule models to onnx model successfully.

## Environment

PyTorch version: 1.4.0+cpu
Is debug build: No
CUDA used to build PyTorch: None

OS: Ubuntu 18.04.4 LTS
GCC version: (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
CMake version: version 3.10.2

Python version: 3.6
Is CUDA available: No
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA

Versions of relevant libraries:
[pip3] numpy==1.18.1
[pip3] torch==1.4.0+cpu
[pip3] torchvision==0.5.0+cpu

## Additional context

cc @suo @houseroad @spandantiwari @lara-hdr @BowenBao @neginraoof

**Comment by suo** [comment]
cc @spandantiwari

**Comment by BynaryCobweb** [comment]
I found the export working with a more recent opset.
Default opset is 9, but if we specify opset_version=11 it works correctly.
```python
torch.onnx._export(loaded_m, x, 'model.onnx', opset_version=11, example_outputs=torch.rand((1, 1000)))
```

**Comment by garymm** [comment]
Seems fixed in PyTorch 1.10.0.
